src/merge.ts

```diff
@@ -213,11 +218,17 @@ export function mergeWithMapping(inFileContent: string, destFileContent: string,
         if (destUnit) {
             const destSource = getSourceElement(destUnit)!;
             const destSourceText = toString(...destSource.children);
-            if (options?.collapseWhitespace ?? true ? collapseWhitespace(destSourceText) !== collapseWhitespace(unitSourceText) : destSourceText !== unitSourceText) {
+            const originTarget = getTargetElement(unit);
+            const originTargetText = originTarget ? toString(...originTarget?.children) : '';
+            const destTarget = getTargetElement(destUnit);
+            const destTargetText = destTarget ? toString(...destTarget?.children) : '';
+            if (options?.collapseWhitespace ?? true ? collapseWhitespace(destSourceText) !== collapseWhitespace(unitSourceText)
```

Comment:
why is this condition changed?

Reply:
I changed the logic to a more simple one and give priority to the destination target nodes